integration-tests/k8s/tiny-cluster.yaml

```diff
@@ -312,8 +304,6 @@ spec:
                 port: 8088
                 protocol: TCP
                 targetPort: 8088
-          selector:
```

Comment:
what was it being used for earlier and why is it removed now?

Reply:
Two things :
- The CRD defination has changed [here](https://github.com/apache/druid/pull/13883/files#diff-0d5340ffc6805b8c7259919bbf0da6adf77e0ee1eea7a4cab0cf5a2159d30546R33). The earlier CRD has been removed, it wasn;t using openAPI specification and was an older version of operator-sdk generated defination. The operator moved to kubebuilder.
- The operator adds selectors based on nodeTypes, user dont need to be aware of what the svc name will be. https://github.com/datainfrahq/druid-operator/blob/master/controllers/druid/handler.go#L1025